ospfd/ospf_dump.c

```diff
@@ -1503,6 +1505,33 @@ DEFUN (no_debug_ospf_te,
   return CMD_SUCCESS;
 }
 
+DEFUN (debug_ospf_trace,
```

Comment:
This is meant for OSPF daemon internal debug activity. This trace filter not just meant for vrf.
 Similar to debug pim trace.